core/src/avm2/globals/flash/text/AntiAliasType.as

```diff
@@ -0,0 +1,6 @@
+package flash.text {
```

Comment:
Does thing file belong to the current PR or added mistakenly?

Reply:
I added this intentionally, to verify that the build script works properly with multiple non-imported classes (e.g. identifying which one is the 'main' class).